src/citrine/exceptions.py

```diff
@@ -84,6 +85,21 @@ class WorkflowNotReadyException(RetryableException):
     pass
 
 
+class PollingTimeoutError(RetryableException):
+    """Polling for an asynchronous result has exceeded the timeout."""
+
+    pass
+
+
+class JobFailureError(RetryableException):
```

Comment:
This definitely isn't retryable; once the job failed it will stay failed.

Reply:
True on that one.

Reply:
Done.